docs/topics/loaders.rst

```diff
@@ -385,12 +385,26 @@ ItemLoader objects
         :param xpath: the XPath to extract data from
         :type xpath: str
 
+        :param stop_on_first_match: when ``xpath`` is a list of selectors, should it
```

Comment:
I wonder whether or not this parameter should be added, since I believe the same effect can be achieved already with `TakeFirst`.

Or are you adding it for performance reasons?

Reply:
Truth be told, I forgot that we could get the same behavior with `TakeFirst`.
Not sure if performance is a problem at all.

My concern was related to semantics.

`loader.add_css('field', ['sel1', 'sel2'])` gives the idea of a composition, `field` is a composition of both `'sel1'` and `'sel2'`.

`loader.add_css('field', ['sel1', 'sel2'], stop_on_first_match=True)` would mean they are fallbacks because of different page layouts.